src/vector/index.js

```diff
@@ -292,6 +403,27 @@ async function loadApp() {
             />,
             document.getElementById('matrixchat')
         );
+        const _localSettings = UserSettingsStore.getLocalSettings();
+        sdk.setLanguage(_localSettings.language);
+        dis.register(onAction);
+        if (!_localSettings.hasOwnProperty('language')) {
+          const language = navigator.languages[0] || navigator.language || navigator.userLanguage;
+          if (language.indexOf("-") > -1) {
+            counterpart.setLocale(language.split('-')[0]);
+            UserSettingsStore.setLocalSetting('language', language.split('-')[0]);
+          } else if (language == 'pt-br') {
```

Comment:
If I understand it right, this line will never be used because
`language.indexOf("-") > 1` if `language == "pt-br"` so the first if would be used - not this more specific one